### PR TITLE
Get rid of shitty upstream and really fix nightly build

### PR DESCRIPTION
**Pull request by imkiva**
Fixes what should be fixed in #389 

image

### CODE REVIEW

**Comment by imkiva** [comment]
image

Yes!

**Comment by imkiva** [comment]
bors merge

File: .github/workflows/move-tag.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+msg="$2"
+
+echo "" > "$HOME/.netrc"
+echo "machine github.com" >> "$HOME/.netrc"
+echo "login $GITHUB_ACTOR" >> "$HOME/.netrc"
+echo "password $GITHUB_TOKEN" >> "$HOME/.netrc"
+echo "machine api.github.com" >> "$HOME/.netrc"
+echo "login $GITHUB_ACTOR" >> "$HOME/.netrc"
+echo "password $GITHUB_TOKEN" >> "$HOME/.netrc"
+chmod 600 "$HOME/.netrc"
```
Comment (ice1000) [inline .github/workflows/move-tag.sh:13]: Woc

File: .github/workflows/nightly-build.yml [hunk 2]
```diff
@@ -1,20 +1,21 @@
 name : nightly
 on :
   push :
-    branches : [ main, auto-release ]
+    branches : [ main, when-i-am-free ]
```
Comment (ice1000) [inline .github/workflows/nightly-build.yml:4]: Lmfao

**Review by ice1000 — COMMENTED** [review]
🤣

**Pull request merged**